src/components/Chat/InputMessageV2.tsx

```diff
@@ -170,6 +357,11 @@ const InputMessageV2 = ({
     );
   };
 
+  const plusButtonStyle = React.useCallback(() => {
+    if (type === SIGNED) return colors.darkBlue;
+    return colors.bondi_blue;
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `plusButtonStyle` function is introduced to determine the color of the plus button based on the type of the chat. This is a good practice as it abstracts away the logic for determining the color, making the code cleaner and easier to read.



<!-- This is an auto-generated comment by OSS CodeRabbit -->